spec.html

```diff
@@ -3236,7 +3236,7 @@ <h2>[[OwnPropertyKeys]] ( )</h2>
             The returned List must not contain any duplicate entries.
           </li>
           <li>
-            The Type of each element of the returned List is either String or Symbol.
+            Each element of the returned List is a Property Key.
```

Comment:
```suggestion
            Each element of the returned List is a property key.
```

This type alias is actually lowercased. /cc @syg A counterexample for Relation I guess.